processing/data_collection/gazette/spiders/rr_boa_vista.py

```diff
@@ -0,0 +1,46 @@
+import datetime as dt
+import json
+
+import scrapy
+from dateparser import parse
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RrBoaVistaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "1400100"
+    name = "rr_boa_vista"
+    allowed_domains = ["boavista.rr.gov.br"]
+    start_urls = ["https://www.boavista.rr.gov.br/diario-oficial"]
+
+    def parse(self, response):
+        options = response.xpath('//*[@id="Periodo"]/optgroup/option/@value')
+        for option in options:
+            data = option.extract()
+
+            url = "".join([self.start_urls[0], "?Periodo=", data])
```

Comment:
Use `w3lib` to include this new parameter to the initial URL (http://w3lib.readthedocs.io/en/latest/w3lib.html#w3lib.url.add_or_replace_parameter)

```
import w3lib.url
url = w3lib.url.add_or_replace_parameter(response.url, 'Periodo', data)
```